src/runner.js

```diff
@@ -4,13 +4,22 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
```

Comment:
nit: can we move this require to the top of the file (before parser). Absolute requires usually come first